twisted/python/filepath.py

```diff
@@ -1529,8 +1529,8 @@ def setContent(self, content, ext=b'.new'):
         with sib.open('w') as f:
             f.write(content)
         if platform.isWindows() and exists(self.path):
-            os.unlink(self.path)
-        os.rename(sib.path, self.path)
+            os.remove(self.path)
```

Comment:
That's not a big deal, I'll change it back.  I took this patch from hawkowl's branch